dist/api_data.py

```diff
@@ -1569,7 +1569,7 @@ def __ge__(self, other):
         stable checkpoint taken prior to a shutdown and \c stable returns the
         most recent \c stable_timestamp set with WT_CONNECTION::set_timestamp.
         See @ref transaction_timestamps''',
-        choices=['all_committed','last_checkpoint',
+        choices=['all_committed','all_durable','last_checkpoint',
```

Comment:
I looked at the ticket and Eric asked for the naming to be something different than `all_durable`. What that request considered?

Reply:
Not yet. I wasn't confident writing this code and anticipated that I might be told to go back to the drawing board.

Since it seems to be going somewhat in the right direction, now's probably a good time to have that discussion. The rename itself should be trivial. I'll leave a comment on the ticket.

Reply:
I followed up on the ticket, but absent someone's great idea, I'm inclined to stick with "all_durable".  It made sense to the replication team, and since we're returning the maximum timestamp that is durable (one less than the first hole), everything up to and including this timestamp is durable.